### ISSUE TITLE
telebot: Bad request: Field "disable_web_page_preview" must be of type Boolean

### ISSUE THREAD

**Issue by kagutsuchii** [tucnak/telebot#51]
Hi,

I tried to implement a simple inline bot, but my generated ArticleResult create an error. This is my code for the ArticleResult an the Result:

```
result := telebot.ArticleResult{"bla", "blubb", "blablubb", telebot.ModeMarkdown, true, true, "", true, ""}
results := []telebot.Result{result}

if err := bot.Respond(query, results); err != nil {
    fmt.Println("ouch:", err)
}
```

I get the error:

```
telebot: Bad request: Field "disable_web_page_preview" must be of type Boolean
```

I tried a bit in your code and found the fault in the `MarshalJSON()` method of `inline_article.go`.
There you create the byte stream for every key value pair with

```
const tpl = `"%s":"%s",`
b.WriteString(fmt.Sprintf(tpl, key, value))
```

but that inserts `""` around the bools. Telegram don't like that. ;)
I test the method with a hardcoded

```
if (key == "disable_web_page_preview") {
    b.WriteString(`"disable_web_page_preview":true,`)
} else {
    b.WriteString(fmt.Sprintf(tpl, key, value))
}
```

and this works! The program now says:

```
telebot: Bad request: Field "hide_url" must be of type Boolean
```

I think it would a better idea to add the `""` to the strings directly and then only use `const tpl = `"%s":%s,``.

**Comment by zoni** [comment]
Hi @kagutsuchii, I recently opened a PR (#50) which (I think) greatly improves support for inline mode and the various results it supports. Would you mind giving my branch a try perhaps? I'm willing to bet it will resolve your issue as well.

**Comment by kagutsuchii** [comment]
Hi @zoni, your pull request looks really cool. Very good work! I hope your PR will be acceppted. Until that I will use your branch. ;)
Unfortunately in your branch there is the same problem. In line 85 of ``inline_article.go`` you also have ``const tpl = `"%s":"%s",` ``. So the second ``"%s"`` puts ``""`` also around a boolean. So I also get the error.

**Comment by tucnak** [comment]
Can anyone but me report whether it's been resolved or not?

**Comment by tucnak** [comment]
To my knowledge this issue has been resolved in #50.
